### PR TITLE
feat(query): add 'Website Azure Active Directory Disabled' for ARM

### PR DESCRIPTION
**Pull request by rogeriopeixotocx**
I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/azureResourceManager/website_azure_active_directory_disabled/query.rego [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+	doc := input.document[i]
+	[path, value] = walk(doc)
+
+	value.type == "Microsoft.Web/sites"
+	not common_lib.valid_key(value, "identity")
+
+	result := {
+		"file": input.document[i].file,
+		"documentId": input.document[i].id,
+		"searchKey": "resources.type={{Microsoft.Web/sites}}.properties",
```
Comment (rafaela-soares) [inline assets/queries/azureResourceManager/website_azure_active_directory_disabled/query.rego:15]: `"searchKey": "resources.type={{Microsoft.Web/sites}}"`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/azureResourceManager/website_azure_active_directory_disabled/query.rego [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	doc := input.document[i]
+	[path, value] = walk(doc)
+
+	value.type == "Microsoft.Web/sites"
+	not common_lib.valid_key(value.identity, "type")
+
+	result := {
+		"file": input.document[i].file,
+		"documentId": input.document[i].id,
+		"searchKey": "resources.type={{Microsoft.Web/sites}}.properties.identity",
```
Comment (rafaela-soares) [inline assets/queries/azureResourceManager/website_azure_active_directory_disabled/query.rego:32]: `"searchKey": "resources.type={{Microsoft.Web/sites}}.identity"`

File: assets/queries/azureResourceManager/website_azure_active_directory_disabled/query.rego [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	[path, value] = walk(doc)
+
+	value.type == "Microsoft.Web/sites"
+	common_lib.valid_key(value.identity, "type")
+	not is_valid_identity(value.identity)
+
+	result := {
+		"file": input.document[i].file,
+		"documentId": input.document[i].id,
+		"searchKey": "resources.type={{Microsoft.Web/sites}}.properties.identity",
```
Comment (rafaela-soares) [inline assets/queries/azureResourceManager/website_azure_active_directory_disabled/query.rego:50]: `"searchKey": "resources.type={{Microsoft.Web/sites}}.identity"`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/azureResourceManager/website_azure_active_directory_disabled/query.rego [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	identity.type == "SystemAssigned"
+} else {
+	identity.type == "UserAssigned"
+	common_lib.valid_key(identity, "userAssignedIdentities")
+}
+
+valid_key(obj, key) {
+	_ = obj[key]
+	not is_null(obj[key])
+}
```
Comment (rafaela-soares) [inline assets/queries/azureResourceManager/website_azure_active_directory_disabled/query.rego:67]: this function should be deleted

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Review by rafaela-soares — APPROVED** [review]
LGTM

**Pull request merged**